internal/node/validate_test.go

```diff
@@ -1,3 +1,8 @@
+// corev1.Endpoints is deprecated, but we still need to use the endpoints api from
+// the validation code since the kubelet kubeconfig/role only has permissions to endpoints
+// and not endpointslices
+//
+//nolint:staticcheck
```

Comment:
i couldnt get golint to respect this directive unless i put it here at the package level. putting it right above the test func or inline with the usage wasnt taking.